roles/network_plugin/calico/rr/tasks/main.yml

```diff
@@ -47,6 +47,25 @@
   notify: restart calico-rr
 
 - name: Calico-rr | Configure route reflector
+  command: |-
+    {{ bin_dir }}/etcdctl \
+    --endpoints={{ etcd_access_addresses }} \
+    --cert={{ etcd_cert_dir }}/admin-{{ groups['etcd'][0] }}.pem \
```

Comment:
Usually we use environment for the keys and certs. See https://github.com/kubernetes-incubator/kubespray/blob/master/roles/etcd/tasks/configure.yml#L12

Reply:
Thanks @woopstar, I've updated accordingly